tests/cephadm/test_cephadm_bootstrap_with_spec_file.py

```diff
@@ -0,0 +1,109 @@
+import yaml
+
+from cli.exceptions import ConfigError, ResourceNotFoundError
+from cli.ops.cephadm import bootstrap, set_container_image_config
+from cli.utilities.configure import (
+    get_tools_repo,
+    setup_client_node,
+    setup_installer_node,
+)
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def create_spec_file(installer, nodes, specs, spec_file):
```

Comment:
Move this method here - https://github.com/red-hat-storage/cephci/blob/master/cli/ops/cephadm.py#L23

and make this part of https://github.com/red-hat-storage/cephci/blob/master/cli/ops/cephadm.py#L24

Reply:
Thanks @vamahaja 
Updated